src/journal-remote/journal-remote.c

```diff
@@ -727,15 +731,23 @@ static int setup_microhttpd_server(RemoteServer *s,
                 goto error;
         }
 
-        r = sd_event_add_io(s->events, &d->event,
+        r = sd_event_add_io(s->events, &d->io_event,
                             epoll_fd, EPOLLIN,
                             dispatch_http_event, d);
         if (r < 0) {
                 log_error_errno(r, "Failed to add event callback: %m");
                 goto error;
         }
 
-        r = sd_event_source_set_description(d->event, "epoll-fd");
+        r = sd_event_add_time(s->events, &d->timer_event,
+                              CLOCK_BOOTTIME, ULONG_LONG_MAX, 5000,
```

Comment:
the time is uint64_t here, hence please use UINT64_MAX instead of ULONG_LONG_MAX.

Reply:
also, unless you have a good reason to, please pass "0" as accuracy, in which case sd-event will figure out a good value for you.

And then, unless you have a good reason to, use CLOCK_MONOTONIC for this, rather than CLOCK_BOOTTIME, since the latter is a lot slower actually (and less widely supported)